src/components/AddItem.js

```diff
@@ -1,11 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { db } from '../lib/firebase';
+
+const initialFormState = {
+  itemName: '',
+  timeFrame: 7,
+  lastPurchased: null,
+};
 
 const AddItem = () => {
+  const token = 'user token';
```

Comment:
Token is just a placeholder right now. I know we discussed having token being passed as a props to AddItem so we could use the user token that's in local storage

Reply:
One thing that you could do is actually go ahead and make it a prop right now. Then, in the parent component you could pass in the mock token
or...better yet, you could play around with `defaultProps` (https://www.codecademy.com/courses/react-101/lessons/this-props/exercises/defaultprops) and make the token be a defaultProp of "user token". Then, when we're ready to work with the real token the code will already be expecting the token to be a prop, you can switch the defaultProps.token to an empty string, and just pass in the real token! Just an idea...

Reply:
While reading about propTypes I ran across documentation for [defaultProps](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values)